apps/android/app/src/main/java/com/axon/app/ui/jobs/JobsViewModel.kt

```
@@ -0,0 +1,103 @@
+                    val r = container.axonRepository.listJobs(kind)
+                    emit(
+                        r.fold(
+                            onSuccess = { Resource.Ready(it) },
+                            onFailure = { Resource.Error(it.message ?: "Error") },
+                        )
+                    )
+                    delay(POLL_INTERVAL_MS)
+                }
+            }
+        }
+        .catch { e -> if (e is CancellationException) throw e; emit(Resource.Error(e.message ?: "Error")) }
```

detekt (1.23.8)

[warning] This else block is empty and can be removed.

(detekt.empty-blocks.EmptyElseBlock)